src/containers/Editor/components.tsx

```diff
@@ -21,6 +21,13 @@ import AccountBottomBar from "components/AccountBottomBar";
 import TransactionBottomBar from "components/TransactionBottomBar";
 import ScriptBottomBar from "components/ScriptBottomBar";
 import { Version } from "components/CadenceVersion";
+import { MdeEditor } from '../../components/MdeEditor';
```

Comment:
Can we use absolute import here?
```suggestion
import { MdeEditor } from 'components/MdeEditor';
```

Reply:
Done. j00lz changed the `MdeEditor`, and after having a final look I realized the `Markdown` component also needed to be changed to absolute import so I've done that. Looks like j00lz added a line to the webpack config, which I don't believe we need, so I removed. If I'm wrong for any reason please let us know @MaxStalker. Marking this resolved.